src/control/server/server_utils.go

```diff
@@ -218,12 +218,12 @@ func prepBdevStorage(srv *server, usr *user.User, iommuEnabled bool, hpiGetter g
 
 	for _, engineCfg := range srv.cfg.Engines {
 		// Calculate mem_size per I/O engine (in MB)
-		engineCfg.MemSize = hugePages.Free / len(srv.cfg.Engines);
-		engineCfg.MemSize *= (hugePages.PageSizeKb >> 10);
-		// TODO: Warn if hugepages are not enough to sustain average
+		engineCfg.MemSize = hugePages.Free / len(srv.cfg.Engines)
+		engineCfg.MemSize *= (hugePages.PageSizeKb >> 10)
+		// Warn if hugepages are not enough to sustain average
 		// I/O workload (~1GB)
-		if ((engineCfg.MemSize / engineCfg.TargetCount) < 1024) {
-			srv.log.Debugf("Not enough hugepages are allocated!")
+		if (engineCfg.MemSize / engineCfg.TargetCount) < 1024 {
+			srv.log.Infof("Not enough hugepages are allocated!")
```

Comment:
sorry my mistake but this should probably be Errorf, don't worry about changing it for this PR but next time you are in the area...